concordium-consensus/src/Concordium/GlobalState/AccountMap.hs

```diff
@@ -68,6 +71,12 @@ newtype AccountMap (pv :: ProtocolVersion) fix = AccountMap {
 -- |The account map to be used in the persistent block state.
 type PersistentAccountMap pv = AccountMap pv BufferedFix
 
+migratePersistentAccountMap
```

Comment:
Since all of these functions are pretty much trivial I do not want to repeat documentation of each and every one of them.
Similar to how we almost never document implementations of typeclass instances.

I will just link to the documentation of `migratePersistentBlockState`.